### ISSUE TITLE
Build 'ogv-encoder.js' with basic Theora, Vorbis encoder interfaces

### ISSUE THREAD

**Issue by bvibber** [brion/ogv.js#56]
For some additional experiments I'm looking into things that could involve client-side transcoding or synthesis of video (say, from an animation). Wrapping an interface similar to the decoder one around the encoders shouldn't be too hard.

**Comment by Kagami** [comment]
Hi. I working on similar project, but it oriented towards WebM (VP8+Opus) encoding. Would be nice if we can share our experience. (I also left comment at your blog.)
https://github.com/Kagami/webm.js
https://github.com/Kagami/ffmpeg.js

**Comment by Hideman85** [comment]
Yes this would be awesome, when we look over internet there is already libraries for doing video only or audio only but none for both.
I'm not an expert at all of webassembly but I guess you have the base for having the possibility of doing encoding too.
And safari wont support webm natively and correctly before a while :wink:
